src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -1358,6 +1358,9 @@ public static Path copy(Path source, Path target, CopyOption... options)
      *     associated with a different provider to this object. </td>
      * </tbody>
      * </table>
+     * If the {@code ATOMIC_MOVE} option is not specified, then the check for
+     * an existing file and the actual move might not be atomic with respect
+     * to other filesystem activities.
```

Comment:
This looks quite good. A small suggestion is to replace "the check for an existing file" with "the check if the target file exists". The reason is that the other sentences use the phrase "target file exists" a few times.

Reply:
Changed in 06732ccbdc4f0e39434eeee51b9a44c2d5a4b6b0.